src/components/BottomSheet/BottomSheetContent/BottomSheetContent.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {memo} from 'react'
+import {View} from 'react-native'
+import type {ITheme} from 'src/theme'
+import styled from 'styled-components/native'
+import type {BottomSheetContentProps} from './types'
+
+const Container = styled(View)<{theme: ITheme}>(({theme}) => ({
+  padding: theme?.sizes?.petite,
+}))
+
+const BottomSheetContentComponent: React.FC<BottomSheetContentProps> = ({style, children}) => (
```

Comment:
why we need `BottomSheetContentComponent`? Can we use direct `Container`?

Reply:
because BottomSheetContent only show content. BottomSheetContentContainer has calculate content height

Reply:
I mean `Container` not `BottomSheetContentContainer`. because the implement for `BottomSheetContentComponent` just render the `Container`

Reply:
okay we will talk about it at meeting